posthog/temporal/tests/data_imports/test_end_to_end.py

```diff
@@ -2118,3 +2120,197 @@ async def test_append_only_table(team, mock_stripe_client):
     # We should now have 2 rows with the same `id`
     assert len(res.results) == 2
     assert res.results[0][0] == res.results[1][0]
+
+
+@pytest.mark.django_db(transaction=True)
+@pytest.mark.asyncio
+async def test_that_activity_triggers_schedule_buffer_one(team: Team, mock_stripe_client):
+    # Setup: create an external data source
+    source = await sync_to_async(ExternalDataSource.objects.create)(
+        source_id=uuid.uuid4(),
+        connection_id=uuid.uuid4(),
+        destination_id=uuid.uuid4(),
+        team=team,
+        status="running",
+        source_type="Stripe",
+        job_inputs={"stripe_secret_key": "test-key", "stripe_account_id": "acct_test"},
+    )
+
+    # Setup: create external data schema
+    schema = await sync_to_async(ExternalDataSchema.objects.create)(
+        name="BalanceTransaction",
+        team_id=team.pk,
+        source_id=source.pk,
+        sync_type=ExternalDataSchema.SyncType.INCREMENTAL,
+        sync_type_config={"incremental_field": "created", "incremental_field_type": "integer"},
+    )
+
+    # Create workflow input
+    inputs = ExternalDataWorkflowInputs(
+        team_id=team.pk,
+        external_data_source_id=source.pk,
+        external_data_schema_id=schema.pk,
+    )
```

Comment:
Any reason we're creating a source/schema/inputs model when we're just testing that the activity calls another function with the identifier passed in?